layouts/partials/head.html

```diff
@@ -18,8 +18,7 @@
 {{ hugo.Generator }}
 
 <!--Favicon-->
-<link rel="shortcut icon" href="{{ site.Params.favicon | absURL }}" type="image/x-icon">

```

Comment:
You should keep this one for Chrome compatibility on Android.
https://github.com/audreyfeldroy/favicon-cheat-sheet#optional-but-encouraged

Reply:
I will use Realfavicongenerator